L81-core-cpp-jwt-token-audience.md

```diff
@@ -0,0 +1,128 @@
+L81: Custom Audience in JWT Access Credentials and Google Default Credentials
+----
+* Author(s): [Jiangtao Li](https://github.com/jiangtaoli2016), [Yihua Zhang](https://github.com/yihuazhang)
+* Approver: markdroth
+* Status: Draft {Draft, In Review, Ready for Implementation, Implemented}
+* Implemented in: Core, C++
+* Last updated: 2021-05-25
+* Discussion at: <google group thread> (filled after thread exists)
```

Comment:
Please create a discussion thread and add a link to the thread here.

Reply:
Here is already a discussion thread. I just use this thread then.

Reply:
Please start a new thread that is specifically pointing to this gRFC.

Reply:
I created a new topic related to this gRFC.